### DISCUSSION TITLE
Add kube-bench self version

### DISCUSSION THREAD

**Discussion by yoavrotems** [aquasecurity/kube-bench#749] [Development]
Hi all !
I thought about adding self version support to kube-bench in order to know what version kube-bench you are running.
The problem is there is already --version flag to mention the kubernetes version, WDYT ? should we change the existing flags? (could harm existing uses in kube-bench) or should we add something like kube-bench-version flag? Do you see the need for it?

**Comment by mozillazg** [comment]
@yoavrotems We already have `kube-bench version`, maybe it's enough.

```
$ ./kube-bench version
v0.4.0
```
